src/zenml/cli/__init__.py

```diff
@@ -959,20 +959,32 @@ def my_pipeline(...):
 zenml artifact list
 ```
 
-The metadata of an artifact can only be deleted if it is no longer linked to
-any pipeline runs, i.e., if the run that produced the artifact and all runs that
-cached any of its steps have been deleted.
+Each artifact has one or several versions. To list artifact versions, use:
 
-To delete all artifacts that are no longer linked to any pipeline runs, use:
+```bash
+zenml artifact versions list
+```
+
+If you would like to rename an artifact or adjust the tags of an artifact or
+artifact version, use the corresponding `update` command:
 
 ```bash
-zenml artifact prune
+zenml artifact update <NAME> -n <NEW_NAME>
+zenml artifact update <NAME> -t <TAG1> -t <TAG2> -r <TAG_TO_REMOVE>
+zenml artifact version update <NAME> -v <VERSION> -t <TAG1> -t <TAG2> -r <TAG_TO_REMOVE>
 ```
 
-To delete a specific artifact, use:
+The metadata of artifacts or artifact versions stored by ZenML can only be 
+deleted once they are no longer used by any pipeline runs. I.e., an artifact
+version can only be deleted if the run that produced it and all runs that used
+it as an input have been deleted. Similarly, an artifact can only be deleted if
+all its versions can be deleted.
+
+To automatically delete all artifacts and artifact versions that are no longer 
```

Comment:
```suggestion
To delete all artifacts and artifact versions that are no longer 
```

Gives the impression that this is some daemon that sits in the background deleting things, otherwise..